src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -97,21 +108,29 @@ public void addPerson(Person p) {
     }
 
     /**
-     * Adds a person to the address book.
-     * The person must not already exist in the address book.
+     * Adds a group to the address book.
+     * The group must not already exist in the address book.
      */
     public void addGroup(Group g) {
         groups.add(g);
     }
 
+    /**
+     * Adds a task into a specified group to the address book.
+     * The group must already exist in the address book.
+     * The task must not already exist in the address book.
```

Comment:
Maybe this would be more accurate: "The task must not already exist in the specified group in the address book."

Please correct me if I'm wrong but same `Task` can exist in the address book as long as it is not contained in the same `Group`?

Reply:
Yeap, your phrasing makes more sense cause same task can exists in different group. will change.